styleguide/source/_patterns/01-atoms/_display-switch.twig

```diff
@@ -0,0 +1,3 @@
+<li class="ama__display-switch">
```

Comment:
The underline at the beginning of this file name means these are not present in the menu? Is this correct? If so, please update the testing instructions.

Reply:
The reasoning is that the orphan `<li>` shouldn't be used on its own (see https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/293#pullrequestreview-129753757) - I don't have strong feelings one way or the other

Reply:
Fair.